monkey/monkey_island/cc/ui/src/components/configuration-components/TextBox.js

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+class TextBox extends React.Component {
+
+  render() {
+    return (
+    <>
+      <p>{this.props.schema.text} </p>
+    </>
```

Comment:
Empty tags can be removed. See [this](https://github.com/guardicore/monkey/pull/1311#discussion_r667164115).